plugins/woocommerce-admin/client/order-attribution/style.scss

```diff
@@ -0,0 +1,61 @@
+.woocommerce-order-attribution-install-analytics-overview-banner {
+
+	margin: 0 15px 10px 0;
+	animation: isLoaded;
+	min-width: 550px; // for smaller screens
```

Comment:
See comment in https://github.com/woocommerce/woocommerce/pull/51672#issuecomment-2373729475

Reply:
Thanks for catching that. I added a media query so the image and the text will be aligned vertically when the screen is smaller.

https://github.com/woocommerce/woocommerce/pull/51672/commits/749fb4dd2a5a4e34933b14b888380db0bc8cf3b3#diff-71e16bc648210308ae9a1f2e30c8e8e311fe24425077449417a2f94ca8b2aa9bR16-R19